File: Projetos-Java/MeuPrimeiroProjeto/app/src/main/java/com/example/meuprimeiroprojeto/MainActivity.java
package com.example.meuprimeiroprojeto;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void aleatorizaNum(View view){

        Random rand = new Random();

        int numRandom = rand.nextInt(11);

        String numRand = Integer.toString(numRandom);

        TextView texto = findViewById(R.id.resultado);
        texto.setText(numRand);

    }
}

File: Projetos-Kotlin/Projetos Android/MotivationApp/app/src/main/java/com/example/motivation/ui/MainActivity.kt
package com.example.motivation.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.motivation.R
import com.example.motivation.infra.SecurityPreferences

class MainActivity : AppCompatActivity() {

    private lateinit var mSecurityPreferences: SecurityPreferences

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mSecurityPreferences = SecurityPreferences(this)
    }
}
